<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1031</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="info">
     <attribute name="title">
      <string>Paper search</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>0</y>
        <width>911</width>
        <height>631</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>論文標題：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_title"/>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_by_title">
             <property name="text">
              <string>依標題關鍵字搜尋</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_by_titlesta">
             <property name="text">
              <string>統計資料</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>作者姓名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Authors"/>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_by_Authors">
             <property name="text">
              <string>依作者關鍵字搜尋</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_by_authorsta">
             <property name="text">
              <string>統計資料</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,1,1">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>論文發表方式：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_type"/>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_by_type">
             <property name="text">
              <string>依標題、作者、發表方式搜尋</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_by_type_sta">
             <property name="text">
              <string>統計資料</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkBox_title">
             <property name="text">
              <string>論文標題</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_abstract">
             <property name="text">
              <string>論文摘要</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_type">
             <property name="text">
              <string>論文型態</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_text">
             <property name="text">
              <string>論文全文</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QComboBox" name="comboBox_page"/>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_prepage">
             <property name="text">
              <string>上一頁</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="p_But_nextpage">
             <property name="text">
              <string>下一頁</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="next_pushButton">
             <property name="text">
              <string>Paper Information&gt;&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="table"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>paper Information</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>961</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_cap">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>論文標題</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_title"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>作者</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_authors"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>摘要</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_abstract"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="pre_pushButton">
             <property name="text">
              <string>&lt;&lt;Back to Search Page</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="actionSave_Data">
             <property name="text">
              <string>save file</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="actionEXIT">
             <property name="text">
              <string>Exit</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_Img">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
